app/src/main/java/org/greenstand/android/TreeTracker/database/AppDatabase.kt

```diff
@@ -58,6 +58,7 @@ abstract class AppDatabase : RoomDatabase() {
                             MIGRATION_4_5,
                             MIGRATION_5_6,
                             MIGRATION_6_7,
+                            MIGRATION_8_9
```

Comment:
Not needed, since we're using the auto migration

Reply:
It was crashing without adding this